src/CONFIG.ts

```diff
@@ -96,6 +96,12 @@ export default {
         IOS_CLIENT_ID: '921154746561-s3uqn2oe4m85tufi6mqflbfbuajrm2i3.apps.googleusercontent.com',
     },
     GCP_GEOLOCATION_API_KEY: googleGeolocationAPIKey,
+    USE_REACT_STRICT_MODE: true,
```

Comment:
We decided to disable strict mode in [this thread](https://expensify.slack.com/archives/C01GTK53T8Q/p1725553387988139?thread_ts=1725376236.233599&cid=C01GTK53T8Q). So we should remove this

Reply:
Updated :)